spec/acceptance/rvm_system_spec.rb

```diff
@@ -111,8 +117,8 @@ class { 'rvm': } ->
 
     it 'reflects installed rubies' do
       shell('/usr/local/rvm/bin/rvm list') do |r|
-        r.stdout.should =~ Regexp.new(Regexp.escape("\n   #{ruby19_version}"))
-        r.stdout.should =~ Regexp.new(Regexp.escape("\n   #{ruby20_version}"))
+        r.stdout.should =~ Regexp.new(Regexp.escape(ruby27_version.to_s))
+        r.stdout.should =~ Regexp.new(Regexp.escape(ruby26_version.to_s))
```

Comment:
Is `.to_s` needed here? Shouldn't they already be strings?

Reply:
yes